src/main/resources/languages/messages_en.yml

```
@@ -146,6 +146,15 @@ messages:
     mode-change: '&b%device% mode changed to: &9%mode%'
     not-shears: '&cA Multi Tool cannot be used as shears!'
 
+  auto-crafting:
+    select-a-recipe: '&eCrouch and Right-Click &7this block with the Item you want it to craft.'
```

the German capitalising `Item`, should be lowercase
```suggestion
    select-a-recipe: '&eCrouch and Right-Click &7this block with the item you want it to craft.'
```

if you really want to capitalise it, then `no-recipes` should alos have it capitalised.